presto-native-execution/presto_cpp/main/http/filters/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@
 add_library(http_filters AccessLogFilter.cpp HttpEndpointLatencyFilter.cpp
                          StatsFilter.cpp)
 
-target_link_libraries(http_filters presto_common ${PROXYGEN_LIBRARIES})
+target_link_libraries(http_filters presto_common ${PROXYGEN_LIBRARIES}
```

Comment:
Is there no proxygen::proxygen target ?

Reply:
I'll add an additional target for the proxygen http server to have more control over linking, then folks can use target names.